types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -2541,6 +2544,13 @@ declare namespace GoogleAppsScript {
             ACCENT6,
             HYPERLINK,
         }
+        /**
+         * An enumeration of the value types returned by Range.getValue and Range.getValues() from the Range class of the Spreadsheet service.
+         * The enumeration values listed below are in addition to Number, Boolean, Date, or String.
+         */
+        enum ValueType {
+            IMAGE = "IMAGE",
```

Comment:
`ValueType.IMAGE` is not defined as string in api documents.

```suggestion
            IMAGE,
```

I tested by running the below code in google apps script:

```js
Logger.log(SpreadsheetApp.newCellImage().valueType); // IMAGE
Logger.log(SpreadsheetApp.ValueType.IMAGE.constructor.name); // Object
Logger.log(SpreadsheetApp.newCellImage().valueType.constructor.name); // Object
```

----

(Optional) If you may want to do it, update the type definitions of `Range#getValue` and `Range#getValues` in SpreadSheetApp:

```ts
getValue(): any;
getValues(): any[][];
```

as

```ts
getValue(): number | boolean | Date | string | ValueType;
getValues(): (number | boolean | Date | string | ValueType)[][];
```